Tasks/DockerComposeV0/utils.ts

```diff
@@ -1,10 +1,35 @@
 "use strict";
+import * as tl from "azure-pipelines-task-lib/task";
 import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+import * as fileutils from "docker-common-v2/fileutils";
 
 export function getFinalComposeFileName(): string {
     return ".docker-compose." + Date.now() + ".yml"
 }
 
 export function writeFileSync(filename: string, data: any, options?: { encoding?: string; mode?: number; flag?: string; }): void {
     fs.writeFileSync(filename, data, options);
+}
+
+function getTaskOutputDir(command: string): string {
+    let tempDirectory = tl.getVariable('agent.tempDirectory') || os.tmpdir();
+    let taskOutputDir = path.join(tempDirectory, "task_outputs");
+    return taskOutputDir;
+}
+
+export function writeTaskOutput(commandName: string, output: string): string {
+    let taskOutputDir = getTaskOutputDir(commandName);
+    if (!fs.existsSync(taskOutputDir)) {
+        fs.mkdirSync(taskOutputDir);
+    }
+
+    let outputFileName = commandName + "_" + Date.now() + ".txt";
```

Comment:
NIT: `.log` file instead?

Reply:
Fixed :), maybe we should fix this in the normal docker task as well?